frontend/src/store/modules/configForm.js

```diff
@@ -134,27 +136,35 @@ export default {
     updateUserAppCfg: (state, userAppCfg) => {
       Object.assign(state.userAppCfg, userAppCfg);
     },
+    setConfigSubmissionInProgress: (state) => {
+      state.configSubmissionSuccess = '';
+      state.configSubmissionError = '';
+      state.configSubmissionInProgress = true;
+    },
     setConfigSubmissionSuccess: (state, msg) => {
       state.configSubmissionSuccess = msg;
       state.configSubmissionError = '';
+      state.configSubmissionInProgress = false;
     },
     setConfigSubmissionError: (state, msg) => {
       state.configSubmissionSuccess = '';
       state.configSubmissionError = msg;
+      state.configSubmissionInProgress = false;
     },
     clearConfigSubmissionMsgs: (state) => {
       state.configSubmissionSuccess = '';
       state.configSubmissionError = '';
     },
-    setGeneratedPassword: function (state, val) {
-      state.generatedPassword = val;
+    setConfigFormSubmissionResult: function (state, val) {
```

Comment:
this one can definitely be an arrow function - no this scope

Reply:
Changed to arrow (as well as the one below)